### ISSUE TITLE
dosemu processes not terminating on exitemu

### ISSUE THREAD

**Issue by severach** [stsp/dosemu2#327]
I run dosemu, and immediately `exitemu`.
````
% ps -ef | grep 'vde'
chris    17483     1  0 16:46 ?        00:00:00 vde_switch -s /tmp/file0eD9Ot
chris    17484     1  0 16:47 ?        00:00:00 sh -c slirpvde -s /tmp/file0eD9Ot --dhcp 2>&1
chris    17485 17484  0 16:47 ?        00:00:00 slirpvde -s /tmp/file0eD9Ot --dhcp
% killall -9 vde_switch slirpvde 
````

**Comment by stsp** [comment]
This is not a dosemu process, this is vde.
Are you sure your vde has all the patches
that dosemu offers for it? If not, dosemu
would write "you have unpatched vde"

**Comment by severach** [comment]
That was it. I made a [package](https://aur.archlinux.org/packages/vde2-dosemu2/) for it.

**Comment by stsp** [comment]
I don't think its correct to do vde2-dosemu2.
The patches are not specific to dosemu2 and
I'd rather wish they integrated in the main vde2
package. I posted all of them into an upstream
tracker but the problem is that it seems the
upstream is not alive.

**Comment by severach** [comment]
dosemu2 is in the name so dosemu2 users can find it. 
A separate package is my only choice. Arch won't integrate the patches unless they are required for functionality or upstream accepts them.
One of the values of Arch is that we users are free to make those decisions without having to wait for someone higher up. We can fix problems now instead of never.

**Comment by severach** [comment]
Am I correct in that this is fixed in virtualsquare/vde-2/ 2.3.3? I don't see any VDE2 messages or orphaned processes.

**Comment by stsp** [comment]
I have no idea.
We use libslirp today.
Please drop vde.
